src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -101,6 +101,16 @@ const LaunchAnalysisModal = ({
       h(ButtonPrimary, { onClick: onDismiss }, ['OK'])
   }, [
     div({ style: { margin: '1rem 0' } }, ['This analysis will be run by ', h(CromwellVersionLink), '.']),
+    div(['Output files will be saved as workspace data in:']),
+    div({ style: { margin: '1rem' } }, [
+      location ? h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [flag]),
+        span({ style: { marginRight: '0.5rem' } }, [regionDescription]),
+        h(InfoBox, [
+          p(['Output (and intermediate) files will be written to your workspace bucket in this region.']),
+          p(['Tasks within your workflow might additionally move or copy the data elsewhere. You should carefully vet the workflows you run if region requirements are a concern.']),
+          p(['Note that metadata about this run will be stored in the US.'])
+        ])]) : 'Loading...'
+    ]),
     div({ style: { margin: '1rem 0' } }, [
       'This will launch ', b([entityCount]), ` analys${entityCount === 1 ? 'is' : 'es'}`,
```

Comment:
If this is the most important information, why not move it to the top instead of the bottom?

Reply:
I guess this is all subjective... but to my brain at least, the "next to the button" location is more visible and my eye is drawn to it more readily than the "right below the title" location.